### ISSUE TITLE
Creating javadocs through `mvn package` causes a Maven build error

### ISSUE THREAD

**Issue by pedrombmachado** [coobird/thumbnailator#175]
## Expected behavior
generation of the jar file without errors

## Actual behavior
```
Building index for all classes...
Generating /home/pedro/repositories/thumbnailator/target/classes/javadoc/allclasses-frame.html...
Generating /home/pedro/repositories/thumbnailator/target/classes/javadoc/allclasses-noframe.html...
Generating /home/pedro/repositories/thumbnailator/target/classes/javadoc/index.html...
Generating /home/pedro/repositories/thumbnailator/target/classes/javadoc/overview-summary.html...
Generating /home/pedro/repositories/thumbnailator/target/classes/javadoc/help-doc.html...
9 errors
36 warnings
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  29.333 s
[INFO] Finished at: 2021-09-02T22:22:04+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.10.1:jar (attach-javadocs) on project thumbnailator: MavenReportException: Error while creating archive: 
[ERROR] Exit code: 1 - /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/resizers/DefaultResizerFactory.java:77: warning: empty <p> tag
[ERROR]  * <p>
[ERROR]    ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/resizers/Resizers.java:38: warning: empty <p> tag
[ERROR]  * <p>
[ERROR]    ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/resizers/Resizers.java:44: warning: empty <p> tag
[ERROR]  * <p>
[ERROR]    ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/Thumbnailator.java:373: warning: no @return
[ERROR] 	public static Collection<File> createThumbnailsAsCollection(
[ERROR] 	                               ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/Thumbnails.java:95: warning: empty <p> tag
[ERROR]  * <p>
[ERROR]    ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/Thumbnails.java:110: warning: empty <p> tag
[ERROR]  * <p> 
[ERROR]    ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/Thumbnails.java:379: warning: empty <p> tag
[ERROR] 	 * <p>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/Thumbnails.java:1169: warning: no @return
[ERROR] 		public Builder<T> sourceRegion(Rectangle region) {
[ERROR] 		                  ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/Thumbnails.java:1211: warning: no @return
[ERROR] 		public Builder<T> crop(Position position) {
[ERROR] 		                  ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/Thumbnails.java:1250: warning: no @return
[ERROR] 		public Builder<T> allowOverwrite(boolean allowOverwrite) {
[ERROR] 		                  ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/Thumbnails.java:1794: warning: empty <p> tag
[ERROR] 		 * <p>
[ERROR] 		   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/Thumbnails.java:1820: warning: empty <p> tag
[ERROR] 		 * <p>
[ERROR] 		   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/Thumbnails.java:2143: warning: empty <p> tag
[ERROR] 		 * <p>
[ERROR] 		   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/ConsecutivelyNumberedFilenames.java:50: warning: empty <p> tag
[ERROR] 	 * <p>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/ConsecutivelyNumberedFilenames.java:89: warning: empty <p> tag
[ERROR] 	 * <p>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/ConsecutivelyNumberedFilenames.java:140: warning: empty <p> tag
[ERROR] 	 * <p>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/ConsecutivelyNumberedFilenames.java:173: warning: empty <p> tag
[ERROR] 	 * <p>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/ConsecutivelyNumberedFilenames.java:235: warning: empty <p> tag
[ERROR] 	 * <p>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/ConsecutivelyNumberedFilenames.java:68: warning: empty <p> tag
[ERROR] 	 * <p>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/ConsecutivelyNumberedFilenames.java:119: warning: empty <p> tag
[ERROR] 	 * <p>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/ConsecutivelyNumberedFilenames.java:205: warning: empty <p> tag
[ERROR] 	 * <p>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/Rename.java:42: warning: empty <p> tag
[ERROR] 	 * <p>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/Rename.java:43: error: tag not allowed here: <dt>
[ERROR] 	 * <dt>Example</dt>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/Rename.java:62: warning: empty <p> tag
[ERROR] 	 * <p>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/Rename.java:63: error: tag not allowed here: <dt>
[ERROR] 	 * <dt>Example</dt>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/Rename.java:102: warning: empty <p> tag
[ERROR] 	 * <p>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/Rename.java:103: error: tag not allowed here: <dt>
[ERROR] 	 * <dt>Example</dt>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/Rename.java:82: warning: empty <p> tag
[ERROR] 	 * <p>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/Rename.java:83: error: tag not allowed here: <dt>
[ERROR] 	 * <dt>Example</dt>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/Rename.java:123: warning: empty <p> tag
[ERROR] 	 * <p>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/Rename.java:124: error: tag not allowed here: <dt>
[ERROR] 	 * <dt>Example</dt>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/Rename.java:165: warning: empty <p> tag
[ERROR] 	 * <p>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/Rename.java:166: error: tag not allowed here: <dt>
[ERROR] 	 * <dt>Example</dt>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/Rename.java:144: warning: empty <p> tag
[ERROR] 	 * <p>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/name/Rename.java:145: error: tag not allowed here: <dt>
[ERROR] 	 * <dt>Example</dt>
[ERROR] 	   ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/filters/Pipeline.java:97: warning: no @param for filter
[ERROR] 	public void add(ImageFilter filter) {
[ERROR] 	            ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/filters/Pipeline.java:108: warning: no @param for filter
[ERROR] 	public void addFirst(ImageFilter filter) {
[ERROR] 	            ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/builders/ThumbnailParameterBuilder.java:45: warning: empty <p> tag
[ERROR]  * <p>
[ERROR]    ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/tasks/io/ImageSink.java:79: warning: no @return
[ERROR] 	public String preferredOutputFormatName();
[ERROR] 	              ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/makers/FixedSizeThumbnailMaker.java:49: warning: empty <p> tag
[ERROR]  * <p>
[ERROR]    ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/makers/ScaledThumbnailMaker.java:37: warning: empty <p> tag
[ERROR]  * <p>
[ERROR]    ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/makers/ScaledThumbnailMaker.java:54: warning: empty <p> tag
[ERROR]  * <p>
[ERROR]    ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/makers/ScaledThumbnailMaker.java:70: error: text not allowed in <DL> element
[ERROR]  * @author coobird
[ERROR]            ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/makers/ScaledThumbnailMaker.java:68: error: element not closed: DL
[ERROR]  * <DL>
[ERROR]    ^
[ERROR] /home/pedro/repositories/thumbnailator/src/main/java/net/coobird/thumbnailator/util/ThumbnailatorUtils.java:96: warning: no @param for format
[ERROR] 	public static List<String> getSupportedOutputFormatTypes(String format) {
[ERROR] 	                           ^
[ERROR] 
[ERROR] Command line was: /home/pedro/.sdkman/candidates/java/8.0.302.fx-zulu/jre/../bin/javadoc @options @packages
[ERROR] 
[ERROR] Refer to the generated Javadoc files in '/home/pedro/repositories/thumbnailator/target/classes/javadoc' dir.
[ERROR] 
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

## Steps to reproduce the behavior
run `mvn package` 

## Environment
java 8 and

pache Maven 3.8.2 

- OS vendor and version:
- Ubuntu 20.04
- JDK vendor and version:
- 8.0.302.fx-zulu
- Thumbnailator version:
- current

**Comment by coobird** [comment]
Thank you for filing this issue, @pedrombmachado!

This is due to Java 8 introducing doclint. Something will be done to fix this.

**Comment by pedrombmachado** [comment]
Thanks @coobird

**Comment by coobird** [comment]
This issue has been corrected in 0.4.15.
